.github/workflows/build.yml

```diff
@@ -0,0 +1,93 @@
+name: build
+
+on:
+  schedule:
+    - cron: '0 10 * * *'  # everyday at 10am
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - 'master'
+
+env:
+  REPO_SLUG_ORIGIN: "moby/buildkit:master"
+  PLATFORMS: "linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64,linux/s390x,linux/ppc64le"
```

Comment:
looks unused

Reply:
I'm a little ahead of schedule :sweat_smile:

Reply:
@tonistiigi Done